src/main/webapp/clinics-map.html

```diff
@@ -0,0 +1,49 @@
+<!DOCTYPE html>
+<html>
+<head>
+  <title>Planned Parenthood Clinic Locations</title>
+  <script src="https://maps.googleapis.com/maps/api/js?key=AIzaSyB1B3SxGUn4uJexYA-8wb5IWSxkPlZyKe8"></script>
+  <script>  
+    function createClinicsMap(){
+      fetch('/clinics-data').then(function(response) {
+        return response.json();
+      }).then((Clinic) => {
+          
+        const map = new google.maps.Map(document.getElementById('map'), {
+          center: {lat: 45.56602083, lng: -94.1503188},
```

Comment:
Could you add a comment as to why you centered at this latlon. It is not clear why you chose that latlon